src/UNL/UCBCN/User.php

```diff
@@ -93,4 +94,11 @@ public function __toString()
     {
         return $this->uid;
     }
+
+    public function getCalendars() {
+        # create options for calendar listing class
+        $options = array('account_id' => $this->account_id);
```

Comment:
I don't think this will get calendars the way you think it will.  I think the `account_id` references the `accounts` table, which is really a set of groups, not accounts.  Different users can have the same `account_id`.  You probably want to use the user's `uid`.